Sourcery/Sourcery.swift

```diff
@@ -235,41 +240,52 @@ public class Sourcery {
         }
     }
 
-    fileprivate func templates(from: Paths) throws -> [Template] {
+    private func templatePaths(from: Paths) -> [Path] {
+        return from.allPaths.filter { $0.isTemplateFile }
+    }
+
+}
+
+#if canImport(ObjectiveC)
+private extension Sourcery {
+    func templates(from: Paths) throws -> [Template] {
         return try templatePaths(from: from).compactMap {
             if $0.extension == "sourcerytemplate" {
                 let template = try JSONDecoder().decode(SourceryTemplate.self, from: $0.read())
                 switch template.instance.kind {
                 case .ejs:
-                    guard EJSTemplate.ejsPath != nil else {
-                        Log.warning("Skipping template \($0). JavaScript templates require EJS path to be set manually when using Sourcery built with Swift Package Manager. Use `--ejsPath` command line argument to set it.")
```

Comment:
@krzysztofzablocki here I was really unsure about this guard statements. Due to changes I have introduced in [EJSTemplate.swift](SourceryJS/Sources/EJSTemplate.swift) file in this MR, `swift test` started to work, but for tests, not for a release build which I am unsure how to verify.

Could you please share some insight on this, shall I leave these guard statements as they were, or would it actually work due to this change I have mentioned? 

Thank you 🙏🏻

Reply:
Nice one, thanks. I'll revert these changes to throw a more precise error as before.

Newer SPM (I think 5.9) supports "resources embedded into binary executables" (see `.embedInCode`) - maybe we could include EJS template contents into code (encoded, say, with base16/64 encoding or as array of bytes) - I'd need to dig deeper into this, but sounds like some form of convenience for Sourcery. I'll revisit this later.